src/sagemaker/feature_store/feature_group.py

```diff
@@ -251,6 +254,9 @@ def wait(self, timeout=None):
             self._processing_pool.close()
             self._processing_pool.clear()
 
+        if not results or results == NotImplementedError:
+            return
```

Comment:
if `results == NotImplementedError`, this would mean that the process did not start or complete successfully. I'm not so sure the real problem has been solved, i.e., ingestion works in py38/39; it looks like we're just swallowing the exception here.

It's possible that the .wait() method (line 243) throws _any_ exception and the `not results` will evaluate to true, since results is initialized to None and never assigned.

Are you sure that `ingest()` actually succeeds?

Reply:
If wait() does timeout, how should we handle ingest() in general? Should we raise an error?